R/update.R

```diff
@@ -218,16 +222,7 @@ get_versions <- function(github_user_repo) {
   gh_urls <- sprintf(base_url, github_user_repo)
 
   unlist(
-    pbapply::pbsapply(
-      gh_urls,
-      function(url) {
-        version <- try(.get_version(url), silent=TRUE)
-        version <- if (inherits(version, "try-error")) NA else version
-        version
-      },
-      USE.NAMES=FALSE
-    ),
+    pbapply::pbsapply(gh_urls, .get_version, USE.NAMES = FALSE),
```

Comment:
@SridharJagannathan I figured one might as well put this error catching logic straight into `.get_version` rather than having an anonymous function as an additional layer.